lighthouse-core/test/audits/uses-rel-preload-test.js

```diff
@@ -25,16 +26,15 @@ describe('Performance: uses-rel-preload audit', () => {
   let mockSimulator;
 
   const mockArtifacts = (networkRecords, mainResource = defaultMainResource) => {
-    return {
+    return Object.assign(Runner.instantiateComputedArtifacts(), {
```

Comment:
this is a bummer :( more incentive to refactor all the compute artifacts I suppose :)